automatic/sql-server-data-tools/legal/VERIFICATION.TXT

```diff
@@ -0,0 +1,16 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+The extension has been downloaded from their official download link listed on <https://marketplace.visualstudio.com/items?itemName=SSIS.SqlServerIntegrationServicesProjects>
+and can be verified like this:
+
+1. Download <https://ssis.gallerycdn.vsassets.io/extensions/ssis/sqlserverintegrationservicesprojects/3.0/1555036193942/Microsoft.DataTools.IntegrationServices.exe>
+2. Then use one of the following methods to obtain the checksum
+  - Use powershell function 'Get-Filehash'
+  - Use chocolatey utility 'checksum.exe'
+
+  checksum type: sha256
+  checksum: F96D1152D442A9FFA46C02D71DBEAED4AFC09A9A4B25564D4AB927FD1BE1D38C
+
+File 'LICENSE.txt' is obtained from <https://marketplace.visualstudio.com/items/SonarSource.SonarLintforVisualStudio/license>
```

Comment:
Wrong link to obtain the license from

Reply:
Have removed the licence file and added a link to the verification.txt